k9mail/src/main/java/com/fsck/k9/view/MessageHeader.java

```diff
@@ -154,10 +185,31 @@ private void onAddSenderToContacts() {
         }
     }
 
+    private void onAddAddressesToClipboard(Address[] addresses){
+        if(mMessage != null){
+            try{
+                String addressesToCopy = "";
+                for(Address addressTemp : addresses){
+                    addressesToCopy += addressTemp.getAddress() + " ";
+                }
+                addressesToCopy = addressesToCopy.substring(0,addressesToCopy.length()-1);
+                ClipboardManager.getInstance(mContext).setText("addresses", addressesToCopy);
+                Toast.makeText(mContext, R.string.copy_address_to_clipboard, Toast.LENGTH_LONG).show();
```

Comment:
It would be nice to use quantity strings here. See http://developer.android.com/guide/topics/resources/string-resource.html#Plurals